src/pack.rs

```diff
@@ -89,6 +90,31 @@ fn load_lockfile(manifest_path: &Path) -> Result<LockFile> {
     })
 }
 
+async fn build_local_package(
+    manifest_path: &Path,
+    output_dir: &Path,
+    platform: &str,
+) -> Result<()> {
+    let manifest_dir = manifest_path
+        .parent()
+        .ok_or_else(|| anyhow!("could not get parent directory of manifest_path"))?;
+    let status = TokioCommand::new("pixi")
```

Comment:
Now that we invoke `pixi` here, I think we should add `pixi` as well in our `pixi.toml` for reproducibility's sake 😅 

When running `pixi run cargo test`, it should then use the pixi from the environment from `PATH` when using `pixi-build`.